pkg/securitycontext/util.go

```diff
@@ -232,3 +237,51 @@ func internalSecurityContextFromPodSecurityContext(pod *api.Pod) *api.SecurityCo
 
 	return synthesized
 }
+
+// AddNoNewPrivileges returns if we should add the no_new_privs option. This will return true if:
+// 1) the container is not privileged
+// 2) CAP_SYS_ADMIN is not being added
+// 3) if podSecurityPolicy.DefaultAllowPrivilegeEscalation is:
+//		- nil, then return true if uid != 0
```

Comment:
This will still break SUID binaries if the uid is set in the Dockerfile rather than through k8s, right?